setup.py

```diff
@@ -26,6 +33,8 @@
         'numpy',
         'PySide2',
         'pyqtgraph>=0.11.0rc0',
+        'pytest-qt',
+        'pytest-timeout',
         'pigpio'
     ],
```

Comment:
Do we need to add 'pytest' here or is that batteries included?

Reply:
i'm actually not sure -- like idk if you even put these in the setup.py or just have a separate `requirements_tests.txt`